apps/openmw/mwmechanics/spellcasting.cpp

```diff
@@ -723,6 +720,11 @@ namespace MWMechanics
                 }
                 else
                     MWBase::Environment::get().getSoundManager()->playSound3D(target, "Open Lock Fail", 1.f, 1.f);
+
+                // Failed attempt is a crime too
+                if (!caster.isEmpty())
+                    MWBase::Environment::get().getMechanicsManager()->objectOpened(getPlayer(), target);
```

Comment:
`objectOpened` is a quite weird name here, especially if we will call it during failed unlock attempts.
Maybe something like `onUnlockAttempt` would be better.

Reply:
That's debatable, but whatever. Renamed.